### PR TITLE
CheckRightsService_AddRightsToDbMigration

### PR DESCRIPTION
**Pull request by mekk1t**
Inserted basic values to database via DbRight configuration class.
Fluent API + .HasData() -> dotnet ef migrations add -> dotnet ef database update

### CODE REVIEW

File: DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs [hunk 1]
```diff
@@ -10,4 +12,25 @@ public class DbRight
...
+                new DbRight
+                {
+                    Id = 1,
+                    Name = "Add/Edit/Remove user",
+                    Description = null
+                },
+                new DbRight
+                {
+                    Id = 2,
+                    Name = "Add/Edit/Remove project",
```
Comment (KostinP) [inline DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: Not sure if this should be one right.

File: DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs [hunk 2]
```diff
@@ -10,4 +12,25 @@ public class DbRight
...
+
+    public class DbRightConfiguration : IEntityTypeConfiguration<DbRight>
+    {
+        public void Configure(EntityTypeBuilder<DbRight> builder)
+        {
+            builder.HasData(
+                new DbRight
+                {
+                    Id = 1,
+                    Name = "Add/Edit/Remove user",
```
Comment (KostinP) [inline DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:24]: Not sure if this should be one right.

**Review by KostinP — CHANGES_REQUESTED** [review]

File: DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs (hunk 1 above)
Comment (KostinP) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: Perhaps another problem is in the name. You can have the right to edit any projects, you can have the right to edit your projects.

Comment (mekk1t) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: I agree, i requested clarificafion from Spartak.
Trello task description states that info, everything in one right.
That also made me think about right types, like, do we need it at all? 
If yes, how we're using it?

**Review by lalagvana — APPROVED** [review]

File: DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs (hunk 1 above)
Comment (lalagvana) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: IMHO in our current model of this service we don't need right types.

Comment (lalagvana) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: Cause now these rights don't relate with specific project/user etc. More general than we realised it at first.

**Comment by mekk1t** [comment]
image
Migration successfully applied

File: DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs (hunk 1 above)
Comment (djino97) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: Unless this should not be all data entered by users (admin) themselves?

Comment (djino97) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: @KostinP  Сan be done multiple rights with one id for one user. Сan be done one right for one user, but with two different id assigned to the one user

Comment (KostinP) [reply DigitalOffice/Services/CheckRightsService/Database/Entities/DbRight.cs:30]: > @KostinP Сan be done multiple rights with one id for one user. Сan be done one right for one user, but with two different id assigned to the one user

I do not argue with this, the question was about something else. I may not understand you.

**Review by KostinP — APPROVED** [review]

**Review by djino97 — APPROVED** [review]

**Review by ifelseelif — APPROVED** [review]

**Review by MihailKulikov — APPROVED** [review]

**Pull request merged**